serverless.ts

```diff
@@ -399,6 +404,30 @@ const serverlessConfig: AWS = {
         EVENT_SOURCE: 'asap.teams',
       },
     },
+    invalidateCache: {
+      handler:
+        'apps/asap-server/src/handlers/invalidate-cache/index-handler.handler',
```

Comment:
index-handler? not sure if the naming is right although i dont actually have a better one

Reply:
I wanted to call it 'apps/asap-server/src/handlers/invalidate-cache/index.handler' but all the files seem to finish with the word handler.

I'll rename it to 'apps/asap-server/src/handlers/invalidate-cache/invalidate-handler.handler'